src/common.js

```diff
@@ -138,7 +138,13 @@ export class EnumSelect extends React.Component {
 
   handleChange(event) {
     this.setState({ value: event.target.value });
-    this.props.onSelect(event.target.value);
+    if (this.props.desc === "format")
```

Comment:
This is a bit hacky, but it's better than being completely broken! 
Could you just add a comment here, something like "TODO: fix this in a nicer way".